typescript/sdk/src/index.ts

```diff
@@ -303,20 +303,30 @@ export {
   defaultViemProviderBuilder,
   protocolToDefaultProviderBuilder,
 } from './providers/providerBuilders.js';
+export {
+  CallData,
+  PopulatedTransaction,
+} from './providers/transactions/types.js';
 export { TxSubmitterInterface } from './providers/transactions/submitter/TxSubmitterInterface.js';
 export { TxSubmitterType } from './providers/transactions/submitter/TxSubmitterTypes.js';
+export {
+  EV5GnosisSafeTxSubmitterPropsSchema,
+  EV5ImpersonatedAccountTxSubmitterPropsSchema,
+} from './providers/transactions/submitter/ethersV5/schemas.js';
 export {
   EV5GnosisSafeTxSubmitterProps,
   EV5ImpersonatedAccountTxSubmitterProps,
-} from './providers/transactions/submitter/ethersV5/EV5TxSubmitterTypes.js';
+} from './providers/transactions/submitter/ethersV5/types.js';
```

Comment:
I raised this question with @yorhodes and @ltyu too:
Is there value in having the schema and types be split across files this way, effectively doubling the file count for this kind of code? It seems to me that they're tightly coupled so there's no reason not to co-locate them.

Reply:
I am 100% on board with this. Faster to develop and fewer files. Perhaps we can rename all files to `zodTypes.ts`

Obviously– issue here is it would be a large refactor. The reason I did it this way is bc we were already doing it this way

@yorhodes wdyt

Reply:
rossy your comment makes sense but in the context of this PR i do think it's good for noah to land it following existing convention, and then we can maybe just do a chore refactor of all of it at once

Reply:
@cmcewen Yep agreed, just meant to propose the change

Reply:
Issue to capture: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3844